whois-update/src/main/java/net/ripe/db/whois/update/authentication/credential/AuthenticationModule.java

```diff
@@ -93,7 +93,7 @@ private boolean hasValidCredentialForCandidate(final PreparedUpdate update, fina
 
             final Class<? extends Credential> credentialClass = credential.getClass();
             for (CredentialValidator credentialValidator : credentialValidatorMap.get(credentialClass)) {
-                if (credentialValidator.hasValidCredential(update, updateContext, offered.ofType(credentialValidator.getSupportedOfferedCredentialType()), credential)) {
+                if (credentialValidator.hasValidCredential(update, updateContext, offered.ofType(credentialValidator.getSupportedOfferedCredentialType()), credential, maintainer)) {
```

Comment:
You could also add the `maintainer` to the `credential` a few lines above, in addition to the auth attribute?

Reply:
Agreed to resolve